packages/processor/source/verifiers/timestamp-verifier.ts

```diff
@@ -18,7 +18,7 @@ export class TimestampVerifier implements Contracts.BlockProcessor.Handler {
 	public async execute(unit: Contracts.BlockProcessor.IProcessableUnit): Promise<boolean> {
 		const result =
 			this.stateService.getStateStore().getLastBlock().data.timestamp <
-			unit.getBlock().data.timestamp + this.configuration.getMilestone().blockTime;
+			unit.getBlock().data.timestamp + this.configuration.getMilestone(unit.height).blockTime;
```

Comment:
I think we can skip height because, configuration height is set to heigh + 1, which should match the current block height.  

```suggestion
			unit.getBlock().data.timestamp + this.configuration.getMilestone().blockTime;
```